### PR TITLE
Add nth term of ap in cpp

### PR DESCRIPTION
**Pull request by niranjantuty**
## Have you read the Contribution Guidelines?

Yes

## Description

Added program to find the nth term of A.P in C-Plus-Plus

## Checklist
- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the README.md and link to my code.
 
 
## Related Issues (Mention Issue no. like #xxx)

Fixes Issue #2 
<br/>

(Write your answer above.)

### CODE REVIEW

File: C-Plus-Plus/README.md [hunk 1]
```diff
@@ -1 +1 @@
-## Add the links to files you have added over here while creating a PR.
\ No newline at end of file
+[Program](https://github.com/sukritishah15/DS-Algo-Point/blob/master/C-Plus-Plus/nth_term_ap.cpp) to find the nth term in the A.P
```
Comment (sukritishah15) [inline C-Plus-Plus/README.md:1]: @niranjantuty Please write this as
[Find nth term in an A.P.](C-Plus-Plus/nth_term_ap.cpp)

Rest is Perfect and good to go!
Also, let me know, once you make the above change.

Comment (niranjantuty) [reply C-Plus-Plus/README.md]: @sukritishah15  I have made the required changes.

Comment (sukritishah15) [reply C-Plus-Plus/README.md]: Perfect @niranjantuty
Merging this PR. 
PR's made on or after Oct. 1 shall contribute to Hacktoberfest.
So, stay tuned, we will be adding more issues!

Comment (niranjantuty) [reply C-Plus-Plus/README.md]: Thank you @sukritishah15 for providing opportunity.

**Pull request merged**